src_py/_sdl3_mixer.py

```
@@ -0,0 +1,105 @@
+
+from pygame import _audio as audio, _sdl3_mixer_c
+
+init = _sdl3_mixer_c.init
+# quit = _sdl3_mixer_c.quit
+get_sdl_mixer_version = _sdl3_mixer_c.get_sdl_mixer_version
+
+
+# Pure Python version of MIX_MSToFrames, since it's so straightforward.
+def ms_to_frames(sample_rate: int, ms: int) -> int:
+    if sample_rate <= 0:
+        raise ValueError("Sample rate must be greater than zero.")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)